integration/operator_test.go

```diff
@@ -98,6 +99,16 @@ func TestMain(m *testing.M) {
 		os.Exit(1)
 	}
 
+	if os.Getenv("VERSION_BUNDLE_VERSION") == "" {
+		version := f.GetVersionBundleVersion(service.NewVersionBundles(), os.Getenv("TESTED_VERSION"))
+		if version == "" {
+			log.Printf("No version bundle version for TESTED_VERSION %q", os.Getenv("TESTED_VERSION"))
+			os.Exit(0)
+		}
+		os.Setenv("VERSION_BUNDLE_VERSION", version)
+	}
+	log.Printf("VERSION_BUNDLE_VERSION %v", os.Getenv("VERSION_BUNDLE_VERSION"))
```

Comment:
I think it would be nice to encapsulate whole that thing in GetVersionBundleVersion.

Reply:
mm not sure, all the code outside the `_test` file is going soon to the `e2e-harness` project so that other projects can use it as a library. I'd prefer to not move code there with side effects, like setting an env var like in this case, even the env var name can depend on the project, WDYT?